index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ohmyfood</title>
+    <meta
+      name="description"
+      content="Ohmyfood, entreprise de commande de repas en ligne"
+    />
+    <link rel="stylesheet" href="./.public/css/style.css" />
+    <script
+      src="https://kit.fontawesome.com/b2f281879d.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!--      HEADER     -->
+
+    <header>
+      <img src="images/logo/ohmyfood.png" alt="logo du site" />
```

Comment:
Don't fortget ./ before" image/..."

Alt must be "Logo de OhMyFood"

Reply:
ok done